vic/drivers/classic/src/vic_force.c

```diff
@@ -0,0 +1,281 @@
+            atmos[rec].vpd[NR] = average(atmos[rec].vpd, NF);
+            atmos[rec].density[NR] = average(atmos[rec].density, NF);
+            atmos[rec].wind[NR] = average(atmos[rec].wind, NF);
+            atmos[rec].snowflag[NR] = false;
+            for (i = 0; i < NF; i++) {
+                if (atmos[rec].snowflag[i] == true) {
+                    atmos[rec].snowflag[NR] = true;
+                }
+            }
+            if (options.LAKES) {
+                atmos[rec].channel_in[NR] =
+                    average(atmos[rec].channel_in, NF) * NF;
+            }
+            if (options.CARBON) {
+                atmos[rec].Catm[NR] = average(atmos[rec].Catm, NF);
+                atmos[rec].fdir[NR] = average(atmos[rec].fdir, NF);
+                atmos[rec].par[NR] = average(atmos[rec].par, NF);
+            }
+        }
+    }
+
+    /****************************************************
+       Variables in the veg_hist structure
+    ****************************************************/
+
+    /* First, assign default climatology */
+    for (rec = 0; rec < global_param.nrecs; rec++) {
+        for (v = 0; v < veg_con[0].vegetat_type_num; v++) {
+            for (i = 0; i < NF; i++) {
+                veg_hist[rec][v].albedo[i] =
+                    veg_lib[veg_con[v].veg_class].albedo[dmy[rec].month -
+                                                         1];
+            }
+        }
+    }
+
+    /* Next, overwrite with timeseries */
+    for (rec = 0; rec < global_param.nrecs; rec++) {
+        for (v = 0; v < veg_con[0].vegetat_type_num; v++) {
+            for (i = 0; i < NF; i++) {
+                uidx = rec * NF + i;
+                if (param_set.TYPE[ALBEDO].SUPPLIED) {
+                    if (veg_hist_data[ALBEDO][v][uidx] != NODATA_VH) {
+                        veg_hist[rec][v].albedo[i] =
+                            veg_hist_data[ALBEDO][v][uidx];
+                    }
+                }
+                if (param_set.TYPE[LAI_IN].SUPPLIED) {
+                    if (veg_hist_data[LAI_IN][v][uidx] != NODATA_VH) {
+                        veg_hist[rec][v].LAI[i] =
+                            veg_hist_data[LAI_IN][v][uidx];
+                    }
+                }
+                if (param_set.TYPE[VEGCOVER].SUPPLIED) {
+                    if (veg_hist_data[VEGCOVER][v][uidx] != NODATA_VH) {
+                        veg_hist[rec][v].vegcover[i] =
+                            veg_hist_data[VEGCOVER][v][uidx];
+                    }
+                }
+            }
+            if (NF > 1) {
+                veg_hist[rec][v].albedo[NR] = average(veg_hist[rec][v].albedo,
+                                                      NF);
+                veg_hist[rec][v].LAI[NR] = average(veg_hist[rec][v].LAI, NF);
+                veg_hist[rec][v].vegcover[NR] = average(
+                    veg_hist[rec][v].vegcover, NF);
+            }
+        }
+    }
+
```

Comment:
If it is broken it should be fixed. 4.2 is a real release and it should work as advertised. We'd just do a bug fix release for it